src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -47,15 +51,25 @@ class ClaimSimulation extends Simulation {
     ninoFormat
   }
 
-  private def getRandomAlphabetCharAsString() = {
-    val randomChar = (random.nextInt(26) + 'A').asInstanceOf[Char]
+  private def getRandomNinoCharAsAString: String = {
+    val randomChar = VALID_NINO_CHARS.charAt(random.nextInt(VALID_NINO_CHARS.length))
     randomChar.toString
   }
 
+  private def getRandomValidTwoNinoCharsAsAString = {
```

Comment:
I tried implementing this with recursion but that kept allowing invalid sequences through. Maybe someone with more Scala knowledge can figure out what's wrong with this code:
```private def getRandomTwoNinoCharsAsAString(): String = {
    val randomChars = getRandomNinoCharAsAString + getRandomNinoCharAsAString
    if (INVALID_NINO_FIRST_TWO_CHARS contains randomChars) {
      println randomChars
      getRandomTwoNinoCharsAsAString()
    }

    randomChars
  }```

Reply:
I don't think anyone on this PR has any more Scala knowledge, so as long as it works I'm ok with it.

Reply:
I don't think the line `getRandomTwoNinoCharsAsAString()` is returning from the method